models/intermediate/int_quickbooks__general_ledger_date_spine.sql

```diff
@@ -21,7 +21,7 @@ with spine as (
 
     {% if execute %}
     {% set last_date_query %}
-        select  max( transaction_date ) as max_date from {{ ref('quickbooks__general_ledger') }}
+        select  coalesce(max(transaction_date), current_date) as max_date from {{ ref('quickbooks__general_ledger') }}
```

Comment:
FYI I did some investigating and could not find any hard confirmation from Postgres, Redshift, Databricks, Snowflake, and BigQuery that confirmed `current_date` would **always** be a date. 

Therefore, to ensure there are **NO** erroneous errors I feel it would be safest to cast `current_date` as date.